reana/reana_dev/cli.py

```diff
@@ -80,6 +80,20 @@ def reana_dev():  # noqa: D301
         $ # example (d): debug mode with code sharing as well
         $ reana-dev cluster-create -m /var/reana:/var/reana --mode debug
 
+    How to setup a local multinode cluster:
+
+    .. code-block:: console
+
+        \b
+        $ reana-dev cluster-create --worker-nodes 3
+        $ kubectl label node kind-worker reana.io/system=infrastructure
+        $ kubectl label node kind-worker2 reana.io/system=runtimebatch
```

Comment:
Not for now, just dumping my mind. Would it be possible to give names to each node so we can avoid `kind-worker2, 3...`?